avm/server/src/runner.rs

```diff
@@ -133,7 +131,7 @@ fn prepare_args(
         IValue::String(air.into()),
         IValue::ByteArray(prev_data.into()),
         IValue::ByteArray(data.into()),
-        IValue::Record(run_parameters),
+        run_parameters,
```

Comment:
From the changes i see in this PR, I do not understand why this change is needed or possible

Reply:
Discussed in private.